### ISSUE TITLE
Feature request：When I send a picture or video from phone to PC，give me an option to delete the source files from my phone?

### ISSUE THREAD

**Issue by nottellyou** [localsend/localsend#1918]
### Is your feature request related to a problem? Please describe.

no

### Describe the solution you'd like

If I choose to delete option, after the file transfer is completed, the transmitted source files will be deleted from my phone, thus saving phone space.

**Comment by Cornmeister** [comment]
+1 :)

**Comment by gritter88** [comment]
+1

**Comment by gritter88** [comment]
it might be a move button.

**Comment by yanxu4780** [comment]
+1

**Comment by Ogekuri** [comment]
+1 💪

**Comment by GsHeri** [comment]
yes PLEASE. +1

how am i supposed to keep track of thousands of files i've sent to my laptop. do you want me to check every single one after the transfer and delete them manually? that's crazy. i remember there IS such an option in the macos/windows version. why not on the phone, where i actually need to free up space?

**Comment by epagone** [comment]
A better solution to this problem is https://github.com/localsend/localsend/issues/2606
Disclaimer: I am the proponent of that feature request

**Comment by GsHeri** [comment]
This is absolutely not a better solution... It's more complexity for an
easily solvable problem

**Comment by Ogekuri** [comment]
I agree, don't sync anything 👌, we just need to automatically delete the uploaded file, so we don't have to search for those files in the phone library anymore.
